packages/heimdalljs/tests/shared-tests/counter-store.ts

```diff
@@ -19,6 +19,6 @@ describe('CounterStore', function() {
       _namespaceCount: 0,
       _nameCache: {},
       _labelCache: {}
```

Comment:
the properties above are `private`, in the `CounterStore` class. As long as they're not public, this POJO will never be type-equivalent to a `CounterStore` or `Partial<CounterStore>`